docs/pages/monitoring/monitor-your-app.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: Monitor your app
+description: Learn how to monitor the usage of your app after its release.
+---
+
+import { ContentSpotlight } from '~/ui/components/ContentSpotlight';
+import { Terminal } from '~/ui/components/Snippet';
+
+Once your app is released, you can track anonymized usage data to give you insights on how users use your app. This data includes which updates are in use, when users experience bugs, and more.
+
+## Expo insights
+
+Expo provides the [`expo-insights`](/eas-insights/introduction/) library, which tracks information related to [EAS Update](/deploy/send-over-the-air-updates/). This data includes the app version, platform, OS version, and update adoption.
+
+You can install `expo-insights` with:
+
+<Terminal cmd={['npx expo install expo-insights']} />
```

Comment:
```suggestion
<Terminal cmd={['$ npx expo install expo-insights']} />
```

Missing `$` prefix in all commands on the page. Let's add it so if there's any syntax highlights applicable, this will cover that.